### PR TITLE
docs - note on locked parameters

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
See https://github.com/metabase/metabase/issues/46375#issuecomment-2263632846

### CODE REVIEW

File: docs/embedding/static-embedding-parameters.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Parameters for static embeds
 redirect_from:
-- /docs/latest/embedding/signed-embedding-parameters
+  - /docs/latest/embedding/signed-embedding-parameters
```
Comment (albertoperdomo) [inline docs/embedding/static-embedding-parameters.md:4]: Is this whitespace intentional? Seems unrelated to the changes

**Review by albertoperdomo — APPROVED** [review]

File: docs/embedding/static-embedding-parameters.md (hunk 1 above)
Comment (jeff-bruemmer) [reply docs/embedding/static-embedding-parameters.md:4]: I formatted the doc with `prettier`.
